01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,29 @@
+const { writeFile, readFile } = require("fs").promises;
+
+const writer = async () => {
+
+  try {
+    await writeFile(
+      './temporary/temp.txt',
+      'Hi\nthere,\nfriends'
```

Comment:
Ideally you would return the await writeFile and have two separate writeFile statements. You don't have to correct but good thing to note.

 await writeFile("temp.txt","Line 1");
  await writeFile("temp.txt","Line 2\n", {flag: "a"});
   var test = await writeFile("temp.txt","Line 3", {flag: "a"});
   return test // where test would be a promise